src/utils.py

```diff
@@ -310,3 +312,47 @@ def get_response_data(json_response, data_default):
     Data default parameters takes what should data to be returned.
     """
     return json_response.get("result", {}).get("data", data_default)
+
+
+def server_run_flow(flow_name, flow_args):
+    """Run a flow.
+
+    :param flow_name: name of flow to be run as stated in YAML config file
+    :param flow_args: arguments for the flow
+    :return: dispatcher ID handling flow
+    """
+    logger.debug('Running flow {}'.format(flow_name))
+    start = datetime.datetime.now()
+
+    init_celery(result_backend=False)
+    dispacher_id = run_flow(flow_name, flow_args)
+
+    # compute the elapsed time
+    elapsed_seconds = (datetime.datetime.now() - start).total_seconds()
+    logger.debug("It took {t} seconds to start {f} flow.".format(
+        t=elapsed_seconds, f=flow_name))
+    return dispacher_id
+
+
+def server_create_analysis(ecosystem, package, version, api_flow=True,
```

Comment:
this would need to be checked as well please

Reply:
Could you elaborate what needs to be checked here? @tisnik